src/app/src/components/EquityMap/EquityMap.tsx

```diff
@@ -0,0 +1,94 @@
+import L from 'leaflet';
+import { useEffect, useState } from 'react';
+import { createPortal } from 'react-dom';
+import { useMap } from 'react-leaflet';
+import StatesLayer from '../StatesLayer';
+
+import UsaMapContainer from '../UsaMapContainer';
+
+export default function EquityMap() {
+    return (
+        <>
+            <div style={{ height: 75 }} />
+            <UsaMapContainer negativeMargin>
+                <StatesLayer />
+                <DisadvantagedCommunitiesLayer />
+            </UsaMapContainer>
+        </>
+    );
+}
+
+function DisadvantagedCommunitiesLayer() {
+    const map = useMap();
+    const [svg, setSvg] = useState<SVGElement | undefined>();
+    const [disadvantagedCommunitiesData, setDisadvantagedCommunitiesData] =
+        useState<GeoJSON.GeoJSON | undefined>();
+
+    useEffect(() => {
+        import('../../data/disadvantagedCommunities.json').then(data => {
+            setDisadvantagedCommunitiesData(data as unknown as GeoJSON.GeoJSON);
```

Comment:
I think this `useEffect` is getting called every time we toggle between the per capita map and then back to the equity map. After the initial toggle to the equity map I would think we wouldn't need to re-render, would `useMemo` be more appropriate here? Also not sure how relevant this is if we're eventually refactoring this as a toggle-able layer.